lecture/views.py

```
@@ -1,49 +1,247 @@
 from django.contrib.auth.decorators import login_required
+from django.views.decorators.csrf import csrf_protect
+from django.utils.decorators import method_decorator
 from django.contrib import messages
 from django.urls import reverse
-from django.http import HttpRequest, HttpResponse
-from .forms import LectureForm, LecturePDFForm
-from .models import Lecture
-from classrooms.models import Classroom
 from django.views.decorators.http import require_http_methods
-
+from django.views.decorators.csrf import csrf_protect
```

Ruff (0.8.2)

Redefinition of unused `csrf_protect` from line 5

Remove definition: `csrf_protect`

(F811)